libraries/AC_PrecLand/AC_PrecLand.h

```diff
@@ -69,6 +69,35 @@ class AC_PrecLand
     // process a LANDING_TARGET mavlink message
     void handle_msg(const mavlink_landing_target_t &packet, uint32_t timestamp_ms);
 
+    // Status of the Target Location
+    enum class PldState: uint8_t {
+        TARGET_LOST = 0,
+        TARGET_OUT_OF_RANGE,
+        TARGET_RECENTLY_LOST,
+        TARGET_FOUND
+    };
+
+    // Strictness that the user wants for Prec Landing
+    enum class PldRetryStrictness: uint8_t {
+        NOT_STRICT = 0,
+        NORMAL,
+        VERY_STRICT
+    };
+
+    // return the last time PrecLand library had a output of the landing target position
+    uint32_t get_last_precland_output_ms() const { return _last_precland_output_ms; }
+
+    // return the current status of the location of the landing target
+    PldState get_plnd_target_status() const { return _current_status; }
+
+    // return the last known landing location in Earth Frame NEU.
```

Comment:
I wonder if we should be using NED - we want to move all of ArduPilot to NED - this isn't going to help